src/System.Diagnostics.DiagnosticSource/src/System.Diagnostics.DiagnosticSource.builds

```diff
@@ -2,8 +2,10 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Diagnostics.DiagnosticSource.csproj" />
+    <Project Include="System.Diagnostics.DiagnosticSource.csproj"/>
```

Comment:
The .csproj file needs to work without wrapping '.builds' wrapper, so you pick the most interesting build variation to build by default (e.g. in VS).   This happens to be netstandard1.3.   Thus the first one is the 'default' one and does not need a TargetGroup.    I could put an explicit TargetGroup here, or a comment, but that has its own potential for mistakes.   I am following conventions used elsewhere when I am not specifying the targetGroup for the first build in the builds file, so I think it is OK as it is...